repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/indicator.ftl

```diff
@@ -10,7 +10,6 @@
   <meta charset="UTF-8" />
   <meta name="title" content="NHS - National Indicator Library" />
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0" />
-  <link rel="stylesheet" href="css/style.css" />
```

Comment:
I will check with you tomorrow why you're not using "base-layout" tpl

Reply:
```yaml
/nationalindicatorlibrary:
        hst:inheritsfrom:
        - ../base
        jcr:primaryType: hst:configuration
```

Reply:
Still inheriting from base, this includes the base-layout template doesn't it? I just noticed that if I took out that stylesheet reference, it made no difference so I removed it